flink-table/flink-table-common/src/main/java/org/apache/flink/table/functions/SqlCallSyntax.java

```diff
@@ -70,6 +70,12 @@ private String doUnParse(
                 }
             };
 
+    /**
+     * Function syntax for functions without parameters (e.g., CURRENT_DATE, LOCALTIMESTAMP,
```

Comment:
Should it be 
```suggestion
     * Function syntax for functions without parenthesis (e.g., CURRENT_DATE, LOCALTIMESTAMP,
```
I'm asking since there are a couple more functions without parameters like `NOW()`, `CURRENT_ROW_TIMESTAMP()` and sometimes `UNIX_TIMESTAMP()` however based on the doc they should have parenthesis
https://nightlies.apache.org/flink/flink-docs-master/docs/dev/table/functions/systemfunctions/

Reply:
Yes, I meant to write parenthesis 😅 Updated it

Reply:
> I'm asking since there is a couple more functions without parameters like NOW(), CURRENT_ROW_TIMESTAMP() and sometimes UNIX_TIMESTAMP() however based on the doc they should have parenthesis
https://nightlies.apache.org/flink/flink-docs-master/docs/dev/table/functions/systemfunctions/

@snuyanzin Could you elaborate on that?